anndata2rdf/src/curated_data/Curated_Lung_Data_KG_V1 - Sheet1.csv

```diff
@@ -0,0 +1,4 @@
+Dataset (indv datasets within larger group),Full name dataset (top of page),CxG Link,h5ad link,Reference_DOI,Bionetworks reference,Standard category present? (T/F),Standard category cell_type present? (T/F),Author Category Cell Type Field Name,Content,Value type(s),Notes,Study Short Name,CxG Dataset Collection X,Is the dataset Normal or Normal/Diseased,Stage
+Human CellCards Multi-Study CellRef 1.0 Atlas,Human CellCards Multi-Study CellRef 1.0 Atlas,https://cellxgene.cziscience.com/e/eb499fd8-7000-419f-8854-926b9b61b11f.cxg/,https://datasets.cellxgene.cziscience.com/443f7fb8-2a27-47c3-98f6-6a603c7a294e.h5ad,DOI: 10.1038/s41467-023-40173-5,T,T,T,"celltype_lev
+el3_fullname",Cell types,Normal,,Guo et al. (2023) Nat Commun,https://cellxgene.cziscience.com/collections/f86d6317-7215-409e-bfda-3f4ded3dadaa,Normal,"Pediatric, Adult"
+An integrated cell atlas of the human lung in health and disease (full),The integrated Human Lung Cell Atlas,https://cellxgene.cziscience.com/e/9f222629-9e39-47d0-b83f-e08d610c7479.cxg/,https://datasets.cellxgene.cziscience.com/b13072bd-9cb6-42ca-9f4f-01252baef273.h5ad,DOI:10.1038/s41591-023-02327-2,T,T,T,ann_finest_level,Cell types,"Normal, abbreviations",,Sikkema et al. (2023) Nat Med,https://cellxgene.cziscience.com/collections/6f6d381a-7701-4781-935c-db10d30de293,Both,Adult
```

Comment:
Same here are some missing author cell types we were using previously;

- ann_finest_level
- ann_level_1
- ann_level_2
- ann_level_3
- ann_level_4
- ann_level_5